crates/rolldown_plugin_reporter/src/lib.rs

```diff
@@ -10,18 +11,21 @@ use std::{
   time::{Duration, Instant},
 };
 
+use cow_utils::CowUtils;
 use rolldown_plugin::{HookUsage, Plugin, PluginContext};
 use sugar_path::SugarPath;
 
 #[derive(Debug)]
+#[allow(clippy::struct_excessive_bools)]
 pub struct ReporterPlugin {
+  assets_dir: String,
+  is_lib: bool,
   is_tty: bool,
   should_log_info: bool,
-  #[allow(dead_code)]
-  chunk_limit: bool,
+  chunk_limit: usize,
   chunk_count: AtomicU32,
   compressed_count: AtomicU32,
-  #[allow(dead_code)]
+  report_compressed_size: bool,
```

Comment:
I'm confusing about this feature? For gb-sized projects, we need to actually do the compressing to report the status. Is it worth? @sapphi-red cc

Reply:
I guess it won't take much time unless the project is enormous. It only compresses the non-binary assets (e.g. JS, CSS). I believe you won't have GBs of JS code; that's not good for the end user as well. Even if it has GBs of JS code in total, you'll probably have multiple chunks and that means it can be parallelized.
Also the user can disable it if preferred.